lib/hooks/blueprints/actionUtil.js

```diff
@@ -115,6 +115,15 @@ module.exports = {
 
     var pk = req.options.id || (req.options.where && req.options.where.id) || req.param('id');
```

Comment:
Why are we using a hardcoded key name of **id**? In general sails assumes the pk would be named "id" but that is not always the case. Should `Model.primaryKey` be used instead?